shoop/themes/classic_gray/templates/classic_gray/shoop/front/basket/partials/_basket_content.jinja

```diff
@@ -28,6 +28,15 @@
                             {{ line.text }}
                         {% endif %}
                     </h4>
+                    {% set package_children = product.get_all_package_children().translated().order_by("translations__name") %}
```

Comment:
Can you check if package parent here? Also can you make macro out of this too since you should probably use this same code in checkout confirmation too?